### ISSUE TITLE
While installing the @nuxt/images package, I had a problem.

### ISSUE THREAD

**Issue by GugoE** [nuxt/image#786]
<img width="782" alt="image" src="https://user-images.githubusercontent.com/24658760/228343519-fd0e7ef7-4301-4b44-860b-2b03c8d56ceb.png">

I first had a problem with Nuxt Fatal Error TypeError: Cannot destructure property 'nuxt' of 'this' as it is undefined.
But then after using:
resolutions: {
     "@nuxt/kit": "3.0.0-rc.13"
   }
I have a new problem:
Cannot read properties of undefined (reading 'find')

PC: Macos M1
Node: v16.16.0
Nuхt 3 version

**Comment by GugoE** [comment]
If someone has problems with Nuxt 3 and @nuxt/images then you need to follow the instructions on this link: https://v1.image.nuxtjs.org/components/nuxt-img
Standard documentation leads to bug with nuxt 3 version

**Comment by danielroe** [comment]
Yes, if you are using Nuxt 3, you should use `@nuxt/image-edge` for now.
